src/gateway-manager/templates/initialize.yaml

```diff
@@ -6,27 +6,44 @@ data:
   initialize.sh: |
     #!/usr/bin/env bash
     set -Eeuo pipefail
+
     gwm="/code/entrypoint.sh"
     $gwm add_app keycloak
-    $gwm add_realm $REALM "Tenant: $REALM" $LOGIN_THEME
-    $gwm add_public_client $REALM $KEYCLOAK_AETHER_CLIENT
-    $gwm add_oidc_client $REALM $KEYCLOAK_KONG_CLIENT
-    $gwm add_user $REALM $KEYCLOAK_INITIAL_USER_USERNAME $KEYCLOAK_INITIAL_USER_PASSWORD
+
+    $gwm add_realm $REALM "Tenant: $REALM" \
+      ${LOGIN_THEME:-keycloak} \
+      ${ACCOUNT_THEME:-keycloak} \
+      ${ADMIN_THEME:-keycloak} \
+      ${EMAIL_THEME:-keycloak}
+
+    $gwm add_public_client $REALM $KEYCLOAK_PUBLIC_CLIENT
+    $gwm add_oidc_client   $REALM $KEYCLOAK_KONG_CLIENT
+
+    # create realm manager
+    $gwm add_admin $REALM $INITIAL_SUPERUSER_USERNAME $INITIAL_SUPERUSER_PASSWORD
+
+    # create users manager
+    $gwm add_user $REALM $INITIAL_ADMIN_USERNAME $INITIAL_ADMIN_PASSWORD
+    $gwm add_user_group $REALM $INITIAL_ADMIN_USERNAME "admin"
+
+    # create simple user
+    $gwm add_user $REALM $INITIAL_USER_USERNAME $INITIAL_USER_PASSWORD
+    $gwm add_user_group $REALM $INITIAL_USER_USERNAME "user"
 
     SOLUTIONS={{ .Values.gw.solutions }}
     for solution in "${SOLUTIONS[@]}"; do
+      if [[ "${RESET_REALM:-}" == "true" ]]; then
+        $gwm remove_solution $solution $REALM
+      fi
```

Comment:
@femilofin this does the trick to reset kong entries :wink:

Reply:
👍